### PR TITLE
Day 1 - Review 

### PR DESCRIPTION
**Pull request by Hariae**
This pull request is not a complete one, completed a basic working code. Need to work more on the problems.

### CODE REVIEW

File: SpecialStack.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    public int pop(){
+        return stack.pop();
+    }
+
+    public int peek(){
+        return stack.peek();
+    }
+
+    public int getMin(){
```
Comment (ashmeetdeol19) [inline SpecialStack.java]: Will this work if you pop top few elements out of the stack ?
Will this function still give minimum value. ?
Think about it.

Comment (Hariae) [reply SpecialStack.java]: I did not run my code against leetcode test cases so ended up missing many edge cases. Made changes to both problems based on inputs from sessions. 

https://github.com/super30admin/Day1/pull/6/commits/4346bce898f79580bd5a988f1c6ceddbc5303551

File: MyHashMap.java [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        System.out.println(myHashMap.get(23));
+        myHashMap.remove(13);
+        System.out.println(myHashMap.get(2));
+        myHashMap.remove(2);
+        System.out.println(myHashMap.get(2));
+        System.out.println(myHashMap.get(13));
+    }
+}
+
+/**
```
Comment (super30admin) [inline MyHashMap.java]: Thats a nice code Hari.
Go for the next two problems.
